README.md

```diff
@@ -53,6 +53,9 @@ API documentation for the most recent release can be found
 To build API documentation for the master branch, check out the
 repository locally and run `rake docs`.
 
+High-level driver documentation including tutorials and the reference that were in the docs folder can now be found
+[here](https://github.com/mongodb/docs-ruby/tree/master/source)
```

Comment:
Probably personal preference, but I think it's good practice to describe what a link points to, so readers (and especially readers who are looking at the formatted text, rather than the raw markdown) can see where the link will take them.

Also, I think it's sufficient to link directly to the repository.

```suggestion
at the docs-ruby repository, [here](https://github.com/mongodb/docs-ruby)
```